### ISSUE TITLE
W: GPG error: https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 Release: 

### ISSUE THREAD

**Issue by trystant** [trystant/dev-error-messages#18]
This error occurred after adding the 3.6 mongo signature according to the [installation instructions](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)

**Comment by trystant** [comment]
```
The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 58712A2291FA4AD5
W: The repository 'https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 Release' is not signed.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by trystant** [comment]
I used the instructions [here](https://askubuntu.com/questions/842592/apt-get-fails-on-16-04-installing-mongodb) to resolve the GPG error.
